src/modules/common/components/bars/topBar/topBar.js

```diff
@@ -62,12 +60,9 @@ const TopBar = ({
       <div className={styles.group}>
         <LightDarkToggle />
         {!isUserLogout && <DiscreteModeToggle />}
-        <Network token={token.active} network={network} t={t} />
-        {isUserLogout && history.location.pathname !== routes.login.path ? (
-          <Link to={routes.login.path} className={styles.signIn}>
-            <PrimaryButton size="s">Sign in</PrimaryButton>
-          </Link>
-        ) : null}
```

Comment:
@isalga please revert this sign in button right now application have braking change, this one is not blong to this ticket